src/installer/installer.go

```diff
@@ -227,6 +265,29 @@ func (i *installer) extractIgnitionToFS(ignitionPath string) (err error) {
 	return err
 }
 
+func (i *installer) patchMastersIgnition(yamlFile, fieldName, newData, op string, yamlStruct interface{}) error {
+	err := i.ops.ReadYaml(yamlFile, yamlStruct)
+	if err != nil {
+		i.log.WithError(err).Error("Failed to read yaml")
+		return err
+	}
+	data := utils.GetFieldFromFieldName(yamlStruct, fieldName)
+	switch op {
```

Comment:
Since we support a single operation (put) and we have a single use case for this.
Also keeping the desire to remove this in the future I wouldn't bother with passing the "put"
I think you can just rename the method to updateManifest or updateYaml.

Reply:
done